src/features/onboarding/OnboardingNavigator.tsx

```diff
@@ -29,6 +33,22 @@ const Onboarding = () => {
         name="AccountEnterPassphraseScreen"
         component={AccountEnterPassphraseScreen}
       />
+      <OnboardingStack.Screen
+        name="AccountSecureScreen"
+        component={AccountSecureScreen}
+      />
+      <OnboardingStack.Screen
+        name="AccountCreatePinScreen"
+        component={AccountCreatePinScreen}
+      />
+      <OnboardingStack.Screen
```

Comment:
we might want to keep these screens out of the swipe back history

Reply:
Do we want them to be able to go back at all after creating a pin? At that point they've finished their initial account setup - we could possibly push a modal for these last few steps or just clear the whole back stack?

Reply:
yeah I don't think they should be able to go back once they've finished this process

Reply:
ok - I'll work that into the next PR